src/routes/reference/robustscale.svx

```diff
@@ -1,11 +1,34 @@
 ---
 title: RobustScale
-blurb: Robust scaling of a FluidDataSet's contents
+blurb: Robust scaling of a DataSet
 tags:
     - data
     - scaling
     - preprocessing
     - statistics
 flair: reference
 category: Analyse Data
----
\ No newline at end of file
+---
+  
+<script>
+    import IndentNote from '$lib/components/IndentNote.svelte';
+</script>
+
+Robust-Scaled data means that each dimension in the data meets two criteria:
+
+1. it has a _median_ of 0 and 
+2. a range of 1 between two specified [percentiles](/reference/bufstats#order-statistics) (these are often the 25th and 75th percentiles). 
+
+RobustScale transforms a [DataSet](/reference/dataset) so it meets the above criteria by analysing the DataSet for the appropriate statistics and scaling each dimension accordingly.
+
+Because RobustScale is based on percentiles, the scaling of the data is less affected by extreme [outliers](/learn/outliers) than other scalers (such as [Standardize](/reference/standardize) and [Normalize](/reference/normalize)). Being more robust to outliers can help RobustScale more accurately scale and represent the majority of the data.
```

Comment:
This is a valuable link to point to in this regard.

https://scikit-learn.org/stable/auto_examples/preprocessing/plot_all_scaling.html